src/wire/causal_snapshot.rs

```diff
@@ -111,11 +118,18 @@ impl<T: AsRef<[u8]>> WireCausalSnapshot<T> {
         }
     }
 
-    /// Return the `count` field
+    /// Return the `epoch` field
     #[inline]
-    pub fn count(&self) -> u32 {
+    pub fn probe_epoch(&self) -> u16 {
```

Comment:
Looks like the getter is returning a `u16`, but the setter is taking a `ProbeEpoch`.
Can they both use `ProbeEpoch`?